installation/macosx.md

```diff
@@ -5,69 +5,76 @@ title: Mac OS X
-
 # Installation on Mac OS X
 
-Please follow these step to get openHAB up and running. You have the choice to use the current stable version or the latest snapshot build.
+This page is structured as follows:
 
-* Download the openHAB distribution as the platform independent `zip` archive, provided at:
-  * Current stable build (recommended): [Download `openhab-2.x.x.zip` here](https://bintray.com/openhab/mvn/openhab-distro/_latestVersion)
-  * Latest snapshot/nightly build: [Download `openhab-2.x.x-SNAPSHOT.zip` here](https://openhab.ci.cloudbees.com/job/openHAB-Distribution).
-* Open the terminal window.
-* Choose a folder name to host your install. The directory will be created in your home folder. Be aware that the path to this folder, and the name of the folder itself, __must not contain any space character__ in order to prevent this classical cause of crash. The name chosen here is `openHAB`.
-* Use the `ls` command in order to check that your directory does not exists yet.
+{::options toc_levels="2..4"/}
 
-```
-cal:~ callas$ ls
-Desktop			Movies
-Documents		Music
-Downloads		Pictures
-Library			Public
-```
+* TOC
+{:toc}
 
-* Move the downloaded file. Assuming that you have downloaded the file in your download folder, the command will be:
+If you're unfamilar with using the Mac OS terminal, then feel free to follow the many guides on the internet. For example:
 
-```
-cal:~ admin$ mv Downloads/openhab-2 openhab
-```
+[Macworld: How to use Terminal on mac](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/)
 
-* Use the "ls" command in order to check that your directory exists now.
+## Installation
 
-```
-cal:~ callas$ ls
-Desktop			Movies	Public
-Documents		Music
-Downloads		openhab
-Library			Pictures
-```
+openHAB is packaged as a platform independent `zip` archive. Installation is as simple as unziping it into a unique folder.
+First, make sure that your system meets the [prerequisites](index.html#prerequisites).
+
+You have the choice to use the current stable version or the latest snapshot build:
+
+* **Stable** 
+  
+  The stable builds have undergone testing, it is the recomended version to download.
+  Download the latest `openhab-2.x.x.zip` [here](https://bintray.com/openhab/mvn/openhab-distro/_latestVersion).
 
-* Move inside the `openhab` folder using the "cd" command.
+* **Snapshot (Unstable)**
+  
+  Snapshots are built daily, and contain the latest changes to openHAB's source code. 
+  These changes haven't undergone the same amount of testing as the stable releases, so only use them if you **need** a new feature.
+  Download the latest `openhab-2.x.x-SNAPSHOT.zip` [here](https://openhab.ci.cloudbees.com/job/openHAB-Distribution).
+  
+The following instructions are performed in a terminal window. At first launch of the terminal, you will be inside your home directory.
+
+Choose a folder name to host your install. Be aware that the path to this folder **must not contain any spaces**. 
```

Comment:
AFAIK, this is a problem with Karaf, which doesn't differentiate between OS for paths.